[DATASTD-2504] Update Whats New and readme for 6.0 (#346)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: dataStandard_versioned_docs/version-6/whats-new/whats-new-v60.md

# What's New - v6.0

Release Date: Nov. 14, 2025.

## Overview

Data Standard v6.0 delivers significant structural and semantic enhancements designed to improve data clarity, interoperability, and reduce update burden across implementations. This release introduces breaking changes and is targeted for deployment beginning in the 2026–27 school year.

Community benefits:

* Reduced Update Burden in StudentEducationOrganizationAssociation--SEOA.

> * Improved Handling of Multiple or Alternative Identifiers.

Reviewer: ```suggestion
* Simplifies the data update process for student identification codes, demographics, and directory information.
* Improved handling of multiple or alternative identifiers.
```

* Enhanced Assessment model to better support data collection and logic for Assessment dashboards.  These changes originated from initiatives in South Carolina, integrating over thirty assessments, and are broadly beneficial for other states.
* Expanded core model coverage for Educator preparator use cases and ongoing updates to related entities.
* Partial handling of Special Education Program events.

> Following reviews by state agencies, SIS and assessment vendors, and feedback from RFC 27 (a, b, c), the following updates were implemented to the Data Standard model in 6.0:

Reviewer: ```suggestion
Following reviews by state agencies, SIS and assessment vendors, and feedback from [RFC 27 - Ed-Fi Data Standard 6.0](https://edfi.atlassian.net/wiki/x/EgC5P), the following updates were implemented to the Data Standard model in 6.0:
```

This single link points to parts a, b, and c.

* Demographic and contact information were decoupled from the SEOA.
* New demographic and directory entities were created for students and staff.  The directory entities contain properties that provide contact information about a student or staff.  This is different than the contact entity which contains parent or representative information.
* The demographic and directory attributes were DELETED from the SEOA and added to the NEW DOMAIN ENTITIES. Note: these specific attributes were not deprecated from the SEOA to avoid redundancy.
* New Optional Identity entities were introduced to better manage multiple identifiers for the same individual. New identity entities were created for students, staff, contact, candidate, and educationorganization.

> * The use of IdentificationCode commons in EducationOrganization, SEOA, and Staff was updated.

Reviewer: ```suggestion
* The StudentEducationOrganizationAssociation and Staff entities were simplified, which reduces the overwrite risk when different systems of record are responsible for a subset of the properties on these entities. In specific, this release makes a breaking change by _moving_:
  - Demographic information to the new StudentDemographic and StaffDemographic entities
  - Contact information to the new StudentDirectory and StaffDirectory entities
  - Identification codes to the new StudentIdentificationCode and StaffIdentificationCode entities
* Because there is a lower risk of multiple systems interacting, the Contact entity has not been modified. However, this release adds a new ContactIdentificationCode entity.
```

This is a significant rewrite. My goals in rewriting were to simplify, provide more rationale, avoid acronyms, and avoid MetaEd specific details. With respect to Metaed: I don't think the reader should need to know or care about the "commons" change. They want to know the effect of that change, not the internal mechanism.

Author: I like the suggested rewrite and approve the changes. Thanks for the suggestions.

* StudentAssessment.AssessedGradeLevel (Optional Descriptor -not a collection) was added to indicate the grade level for which the test form was assessed for the student during the administration. Note: StudentAssessment.WhenAssessedGradeLevel (Optional Descriptor) remains to represent the student’s actual grade level at the time of assessment.
* StudentAssessment.StudentAssessmentIndicator - new optional collection was added to capture vendor-specific context unrelated to scores.  This eliminates the need to misuse assessmentReportingMethodDescriptors or scoreResults for non-score flags.
* ObjectiveAssessment.ParentObjectiveAssessment changed from Optional to Optional Collection.  This allows for an objective assessment to be linked to multiple parent objective assessments.

Details for these changes are found in [Ed-Fi RFC 27c](https://edfi.atlassian.net/wiki/spaces/rc/pages/1229094913/Ed-Fi+RFC+27+c+-+Assessment+Model+Changes)

### Migration from EPDM to Core Entities

Entities and descriptors from EPDM community have been moved into the Core Ed-Fi data standard, promoting consistency and broader applicability.

#### Updates on Special Education Program Association

> [DATASTD-](link)

Reviewer: ```suggestion
[DATASTD-2400](https://edfi.atlassian.net/browse/DATASTD-2400)
```

New optional attributes to the Student
Special Education Program Association to allow the community members collect

> data related to events.

Reviewer: There's actually a breaking change in here.

```suggestion
Two optional IEP date fields have been renamed (BREAKING) to provide greater clarity on their purposes, and two new optional date field have been added.
```

| Property | Cardinality | Definition |
| --- | --- | --- |
| SpecialEducationExitDate | optional | The month, day and year on which a person stops receiving special education services. |
| SpecialEducationExitReason | optional | The reason why a person stops receiving special education services. |
| SpecialEducationExitExplained | optional | Explanation on why a person stops receiving special education services. |

> ### Adding Student Program Evaluation

Author: Same as before: this is an old header.  Need to delete it.

## Minor Changes

### Updates on XXX Field

Description

### Updates on XXX

> [DATASTD-XXX](https://tracker.ed-fi.org/browse/DATASTD-XXXX)

Author: Thanks for the suggestion.  Given your expertise with previous releases, we will remove the list of tickets.

* StudentCharacteristic
* SupporterMilitaryConnection
* TribalAffiliation

#### Fields to New StudentDirectory Entity

* Address
* ElectronicMail
* InternationalAddress
* Telephone

### Deleted StaffEducationOrganizationContactAssociation

As discussed during the DataStandard Work group calls and in alignment with the greater effort to simplify the model. Staff is being split apart into StaffIdentificationCode, StaffDemographic, and StaffDirectory (i.e. contact information such as address, email, telephone) entities. With the creation of targeted individual domain entities, the need to maintain a separate duplicate association for tracking Staff members information has been removed.

Information previously tracked on the Staff domain entity has been split apart in a manner substantially similar to the Student entities above.

#### New Staff Related Entities

* StaffDirectory
* StaffIdentificationCode
* StaffDemographic

### Assessment Model Changes

[DATASTD-2491](https://tracker.ed-fi.org/browse/DATASTD-2491)
[DATASTD-2492](https://tracker.ed-fi.org/browse/DATASTD-2492)
[DATASTD-2493](https://tracker.ed-fi.org/browse/DATASTD-2493)

There were 5 key changes done to assessments:

> * Assessment.academicSubject changed from _Required Descriptor_ to **Required Single Attribute**.

Author: Change done

* ObjectiveAssessment.ParentObjectiveAssessment changed from _Optional_ to **Optional Collection**.  This allows for an objective assessment to be linked to
    multiple parent objective assessments.
* StudentAssessment.SchoolYear changed from _Optional_ to **Required**
* StudentAssessment.AssessedGradeLevel - **Optional Descriptor** was added to indicate the grade level for which the test form was assessed for the
    student during the administration. Note: StudentAssessment.WhenAssessedGradeLevel (Optional Descriptor) remains to represent the student’s actual grade level at the time of assessment.
* StudentAssessment.StudentAssessmentIndicator - **New Optional Collection** was added to capture vendor-specific context unrelated to scores.  This eliminates
    the need to misuse assessmentReportingMethodDescriptors or scoreResults for non-score flags.

:::info

Additional details for these changes can be found in [Ed-Fi RFC 27c](https://edfi.atlassian.net/wiki/spaces/rc/pages/1229094913/Ed-Fi+RFC+27+c+-+Assessment+Model+Changes).

:::

### Migration from Educator Preparation Data Model to Core Entities

> The Educator Preparation Data Model represents a state approved  path or course of study, culminating in a candidate completing all requirements necessary to obtain a certification or licensure to teach within K-12 schools. Individuals under this domain may often be represented in dual roles as students in one context of their continuing education while also being staff at the same time as part of their required training. This domain has largely been held as a community extension until now. Entities and descriptors from Educator Preparation Data Model community have been moved into the Ed-Fi Core data standard in order to promote consistency and broader applicability.

Reviewer 2: Suggest changing "from Educator Preparation Data Model community" to "from Educator Preparation Data Model (Community Edition)"

Author: Implemented

### Update Length Of CourseTitle

[DATASTD-2446](https://tracker.ed-fi.org/browse/DATASTD-2446)

Several community members reached out with the request to increase the overall length of this field. The following domain entities appear to benefit from this expanded functionality.

* Course.CourseTitle (as required)
* CourseOffering.CourseTitle (as optional)
* CourseTranscript.CourseTitle (as optional)
* LearningStandard.CourseTitle (as optional)

### Increase ContentStandardTitle from 75 To 100 Characters

[DATASTD-2468](https://tracker.ed-fi.org/browse/DATASTD-2468)

Received information from vendors regarding tools that allow them to import learning standards data into the API but initially had a misalignment regarding field lengths. In order to promote ease of use and increase consistency the following adjustments were made:

* Updated shared string for ContentStandardTitle to allow 100 characters.
* Updated shared string for Author to allow 255 characters
* Updated CourseTitle to allow 120 characters

> * Updated IdentificationCode to allow 100 characters.

Author: @stevenarnold-ed-fi these two bullet updates were redacted completed. Please review.

### Add "EconomicDisadvantage" Optional Descriptor to StudentDemographic

[DATASTD-2473](https://tracker.ed-fi.org/browse/DATASTD-2468)

The Ed-Fi standard originally had a boolean used by several domain entities in the system to provide insight to those who were under financial hardship. However, as pointed out by members of the community the absence of a flag should not automatically imply that the student/candidate/individual in question does not actually possess that characteristic. By shifting the current boolean value to a descriptor this would provide greater flexibility in allowing the system to capture Yes/No/Unknown values and better populate key state reports.

> ### Replace EconomicDisadvantaged Bool With New EconomicDisadvantaged Descriptor

Author: @stevenarnold-ed-fi The two points for "EconomicDisadvantaged" were redacted as one.

  * CandidateIdentificationCode
  * ContactIdentificationCode
  * EducationOrganizationIdentificationCode
  * StaffIdentificationCode
  * StudentIdentificationCode
* Consequently, the StudentIdentificationCode (_optional_ common) was removed from the SEOA and became its own entity.

* The Assessment, ObjectiveAssessment and StudentAssessment entities were updated (no key structural changes were introduced).

* Five new domains were created to host the EPDM (Community version) entities and descriptors.  Existing Staff and Enrollment domains were enhanced to incorporate EPDM-related entities. In addition, EPDM properties previously marked as deprecated were removed.  See details below.

* Two IEP date fields from the StudentSpecialEducationProgramAssociation were renamed, and two optional attributes were added to improve flexibility.

:::info

Additional details related to the changes for identification and demographics can be found in [Ed-Fi RFC 27b](https://edfi.atlassian.net/wiki/spaces/rc/pages/998375429/Ed-Fi+RFC+27+b+-+Streamlining+Access+to+Identification+Codes+Contact+Information+and+Demographics).

> Attributes decoupled from SEOA were _removed from the SEOA_, rather than deprecated, and _migrated to the new demographic and directory entities_. This approach was reviewed in multiple governance meetings hosted by the Ed-Fi Alliance and endorsed to promote consistency in data standard versioning and to reduce data redundancy.

Reviewer: We need to avoid using acronyms when possible as a best practice.

```suggestion
Attributes decoupled from StudentEducationOrganizationAssociation, Staff, and other entities were _removed _, rather than deprecated, and _migrated_ to the new demographic and directory entities_ This approach was reviewed in multiple governance meetings hosted by the Ed-Fi Alliance and endorsed to promote consistency in data standard versioning and to reduce data redundancy.
```

|------------|-----------|
| `"results": [ { "rating": 999.999, "ratingResultTitle": "string", "resultDatatypeTypeDescriptor": "string" } ]` | `"results": [ { "numericRating": 999.999, "ratingResultTitle": "string", "resultDatatypeTypeDescriptor": "string" } ]` |

### Replaced GenderDescriptor with GenderIdentity

Changes made to the following entities: ApplicantProfile, Staff, Candidate, RecruitmentEventAttendance.

### Updated the Ed-Fi descriptor namespace from "tpdm" to "ed-fi"

Previously, the canonical Ed-Fi descriptor set for EPDM included “tpdm” in their namespace URI values. This is no longer necessary.

For example: uri://tpdm.ed-fi.org/CertificationRouteDescriptor becomes uri://ed-fi.org/CertificationRouteDescriptor

### Removed previously marked deprecated properties

* CandidateEducatorPreparationProgramAssociation.MajorSpecialization
* CandidateEducatorPreparationProgramAssociation.MinorSpecialization
* CandidateIndicator.Period
* Candidate.OldEthnicityDescriptor
* Candidate.CohortYear
* Candidate.Aid
* Candidate.ProgramComplete
* Candidate.DegreeSpecialization
* Candidate.ApplicationReference
* EducatorPreparationProgram.EducatorPreparationProgramType

> ### Removed ProgramGateway Descriptor from PerformanceEvaluation

Reviewer: ```suggestion
### Removed ProgramGateway Descriptor
```
